Makefile

```diff
@@ -54,6 +54,10 @@ bin/k8s:
 bin/registryv1:
 	CGO_ENABLED=0 go build -o $@ ./provisioner/registryv1
 
+bin/unpack:
+	CGO_ENABLED=0 go build -o $@ ./cmd/unpack/...
```

Comment:
nit: should all these `bin/` directives use the `GO_BUILD` variable defined above? ie
`CGO_ENABLED=0 $(GO_BUILD) -o $@ ./cmd/unpack/...`

Reply:
I wasn't able to get that working when I was playing around with it yesterday due to parsing issues. When the verbosity flag is enabled, the go build command is prefixed with a `@` symbol, so that expression gets evaluated as `CGO_ENABLED=0 @go build -o ...` which make can't parse. The alternative was to override that GO_BUILD variable everytime it gets used which didn't feel super ideal either. I think we need to overhaul the Makefile at some point in the near future, so maybe we can visit this conversation then?